cmd/node/node.go

```diff
@@ -941,7 +941,6 @@ func (app *App) LoadOrCreateEdSigner() (*signing.EdSigner, error) {
 		log.With().Info("created new identity", edSgn.PublicKey())
 		return edSgn, nil
 	}
-
 	edSgn, err := signing.NewEdSignerFromBuffer(data)
```

Comment:
are you waiting for the genesis confg to merge before you can initialize the signer?

Reply:
yes, i added a placeholder genesis id initially but then realized that it is better to rebase after config is merged